src/main/java/edu/curssor/moviesPlatform/entities/Movie.java

```diff
@@ -0,0 +1,25 @@
+package edu.curssor.moviesPlatform.entities;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import edu.curssor.moviesPlatform.enums.Category;
+import lombok.Data;
+
+@Data
+public class Movie {
+
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
+    private String id;
+
+    private String name;
+
+    private String category;
+
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
+    private Category categoryDB;
```

Comment:
rename this to:
enum - category
string - categoryLabel

Reply:
Fixed